via/CMakeLists.txt

```diff
@@ -66,21 +66,14 @@ elseif (MSVC)
     target_compile_definitions(vkvia PRIVATE _CRT_SECURE_NO_WARNINGS _USE_MATH_DEFINES)
 endif()
 
-find_package(jsoncpp CONFIG)
-
-if (TARGET jsoncpp_static)
-    # RPATH causes issues for the Linux SDK tarball due to absolute paths.
-    # This can be worked around with relative RPATHS but the tarball doesn't ship jsoncpp.
-    # So the easiest solution is to just the static version of jsoncpp.
-    target_link_libraries(vkvia PRIVATE jsoncpp_static)
-    set_target_properties(vkvia PROPERTIES SKIP_BUILD_RPATH TRUE)
-elseif(UNIX)
-    # https://github.com/LunarG/VulkanTools/issues/1908
-    find_package(PkgConfig REQUIRED)
+# https://github.com/LunarG/VulkanTools/issues/1908
+find_package(PkgConfig)
+if (PKG_CONFIG_FOUND)
     pkg_check_modules(jsoncpp REQUIRED IMPORTED_TARGET jsoncpp)
     target_link_libraries(vkvia PRIVATE PkgConfig::jsoncpp)
 else()
-    message(FATAL_ERROR "Unable to find jsoncpp!")
+    find_package(jsoncpp CONFIG REQUIRED)
+    target_link_libraries(vkvia PRIVATE jsoncpp_static)
```

Comment:
Internal Windows CI doesn't have package config. Furthermore most Windows developers don't tend to have this by default. So for now we support both paths. At least both paths are actively tested though which was the main issue before.